packages/reactivity/__tests__/computed.spec.ts

```diff
@@ -151,6 +151,14 @@ describe('reactivity/computed', () => {
     expect(n.value).toBe(-1)
   })
 
+  it('should support passing object with only one get property', () => {
+    const a = computed(() => 1)
```

Comment:
I think the scenario tested by `a` is already tested by `it('should be readonly'`. I think that part can be removed here.

I also wonder whether the test for `b` should just be merged into `it('should be readonly'`? Or, if not, maybe move this test case to be next to that one?